test/lib/tasks/createDebugFiles.js

```diff
@@ -1,7 +1,6 @@
 const test = require("ava");
 
-const ui5Builder = require("../../../");
-const tasks = ui5Builder.builder.tasks;
```

Comment:
This indirectly also tested that the task is exported via index.js
What about a new test to check for all the exports?

Reply:
Added tests in all repositories